packages/datadog_flutter_plugin/lib/src/datadog_configuration.dart

```diff
@@ -175,24 +177,42 @@ class RumConfiguration {
   // Either a RUM Application Id. Obtained on the Datadog website.
   String applicationId;
 
+  /// Sets the sampling rate for RUM Sessions.
+  ///
+  /// This property is deprecated in favor of [sessionSampleRate]
+  @Deprecated('Use sessionSampleRate instead')
+  double get sampleRate => sessionSampleRate;
+  set sampleRate(double value) => sessionSampleRate = value;
+
   /// Sets the sampling rate for RUM Sessions.
   ///
   /// The sampling rate must be a value between `0.0` and `100.0`. A value of
   /// `0.0` means no RUM events will be sent, `100.0` means all sessions will be
-  /// kept
+  /// sent
+  ///
+  /// Defaults to `100.0`.
+  double sessionSampleRate;
```

Comment:
very nit: wording is a big different between `sessionSampleRate` and `tracingSamplingRate`, we should choose between `sample` and `sampling` for the consistency.

Reply:
Ugh good call. Thanks!